hack/components/bump-linux-bridge.sh

```diff
@@ -27,10 +27,23 @@ echo 'Build container image with linux-bridge binaries'
 LINUX_BRIDGE_TAR_CONTAINER_DIR=/usr/src/github.com/containernetworking/plugins/bin
 LINUX_BRIDGE_IMAGE=quay.io/kubevirt/cni-default-plugins
 LINUX_BRIDGE_IMAGE_TAGGED=${LINUX_BRIDGE_IMAGE}:${LINUX_BRIDGE_TAG}
-(
-    cd ${LINUX_BRIDGE_PATH}
+DOCKER_BUILDER="${DOCKER_BUILDER:-linux-bridge-docker-builder}"
+# By default, the build will be based on the host architecture.
+# To build for other platforms, you can:
+# 1. Export all supported platforms: export PLATFORMS=all
+# 2. Or specify specific platforms: export PLATFORMS=linux/amd64,linux/arm64
+PLATFORM_LIST="linux/amd64,linux/s390x,linux/arm64"
+ARCH=$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
+[ -z "$PLATFORMS" ] && PLATFORMS="linux/${ARCH}"
```

Comment:
please use default value bash syntax, as in line 30

Reply:
Done.